packages/gatsby-theme-newrelic/src/components/ExternalLink.js

```diff
@@ -3,26 +3,40 @@ import PropTypes from 'prop-types';
 import useInstrumentedHandler from '../hooks/useInstrumentedHandler';
 import { localizeExternalLink } from '../utils/localization';
 import useLocale from '../hooks/useLocale';
+import useTessen from '../hooks/useTessen';
+import { useLocation } from '@reach/router';
 
 const isNewRelic = (to) => to.startsWith('https://newrelic.com');
+const isSignup = (to) => to.startsWith('https://newrelic.com/signup');
 
-const ExternalLink = ({ children, onClick, href, ...props }) => {
+const ExternalLink = ({ children, onClick, href, component, ...props }) => {
```

Comment:
Rather than a `component` prop, could you instead make this an `instrumentation` prop to match some of our other components like the [`CreateIssueButton`](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/CreateIssueButton.js#L61-L63)? Using `instrumentation` better calls out the purpose of the prop so that is clear `component` is used for instrumentation purposes. Without it, this appears to change the component that is used to render the link itself:

```
<ExternalLink component="Something" /> // <-- looks like "Something" is supposed to be the component rendered here
```

Reply:
Yeah I was just changing this haha, which is why I hadn't posted it yet! I was making it `trackingProps` that could either be a string or an object of additional properties.

Reply:
Could we standardize on `instrumentation`? If we go with `trackingProps` could we then update the other components so we can settle on a pattern for this? That would be super helpful if we could.

Reply:
it has been now changed to `instrumentation`